File: client/iih_stub/iih.ci_stub/iih.ci.mrqc/outqapatlist/i/IOutQaPatListQryService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using iih.ci.mrqc.outqapatlist.d;
using iih.ci.mrqc.qrydto.d;
using xap.mw.core.data;
using xap.sys.appfw.tmpl.qryscheme.qrydto;

namespace iih.ci.iih.ci.mrqc.outqapatlist.i
{
    public interface IOutQaPatListQryService
    {
        //获取门诊质控患者列表
        PagingRtnData<OutQaPatListDTO> getOutQaPatListDtos(QryRootNodeDTO qryRootNodeDTO, PaginationInfo paginationInfo);
        //获取门诊抽查患者列表
        PagingRtnData<OutQaPatListDTO> getOutQaToRandomPatListDtos(OutQaPatQryDTO qryDTO, PaginationInfo paginationInfo);
        //获取门诊已抽查患者列表
        PagingRtnData<OutQaPatListDTO> getOutQaHasRandomPatListDtos(OutQaPatQryDTO qryDTO, PaginationInfo paginationInfo);
        //获取门诊待审核患者列表
        PagingRtnData<OutQaPatListDTO> getOutQaTracePatListDtos(OutQaPatQryDTO qryDTO, PaginationInfo paginationInfo);
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mrqc/outqapatlist/i/IOutQaPatListQryService.java
package iih.ci.mrqc.outqapatlist.i;

import iih.ci.mrqc.outqapatlist.d.OutQaPatListDTO;
import iih.ci.mrqc.qrydto.d.OutQaPatQryDTO;
import xap.mw.core.data.BizException;
import xap.sys.appfw.orm.handle.dataobject.paging.PaginationInfo;
import xap.sys.appfw.orm.handle.dataobject.paging.PagingRtnData;
import xap.sys.appfw.tmpl.qryscheme.qrydto.QryRootNodeDTO;

public interface IOutQaPatListQryService {
    /**
	*  获取门诊质控患者列表
	*/
    public abstract  PagingRtnData<OutQaPatListDTO> getOutQaPatListDtos(QryRootNodeDTO qryRootNodeDTO,PaginationInfo paginationInfo) throws BizException;

    /**
	*  获取门诊抽查患者列表
	*/
    public abstract  PagingRtnData<OutQaPatListDTO> getOutQaToRandomPatListDtos(OutQaPatQryDTO qryDTO,PaginationInfo paginationInfo) throws BizException;
    /**
	*  获取门诊抽查患者列表
	*/
    public abstract  PagingRtnData<OutQaPatListDTO> getOutQaHasRandomPatListDtos(OutQaPatQryDTO qryDTO,PaginationInfo paginationInfo) throws BizException;
    /**
	*  获取门诊抽查患者列表
	*/
    public abstract  PagingRtnData<OutQaPatListDTO> getOutQaTracePatListDtos(OutQaPatQryDTO qryDTO,PaginationInfo paginationInfo) throws BizException;
}
